src/Cache/MongoLock.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace MongoDB\Laravel\Cache;
+
+use Illuminate\Cache\Lock;
+use MongoDB\Laravel\Collection;
+use MongoDB\Operation\FindOneAndUpdate;
+use Override;
+
+use function random_int;
+
+final class MongoLock extends Lock
+{
+    /**
+     * Create a new lock instance.
+     *
+     * @param  Collection  $collection              The MongoDB collection
+     * @param  string      $name
+     * @param  int         $seconds
+     * @param  string|null $owner
+     * @param  array       $lottery                 The prune probability odds
+     * @param int         $defaultTimeoutInSeconds The default number of seconds that a lock should be held
```

Comment:
```suggestion
     * @param int          $defaultTimeoutInSeconds The default number of seconds that a lock should be held
```

Are all of the `@param` annotations necessary? Elsewhere, I've only seen them used when a description of the parameter is necessary. In that case, you'd only need to do it for `$lottery` and `$defaultTimeoutInSeconds` (the `$collection` description doesn't seem necessary).

Reply:
In fact, here I've been able to add the types to the parameters so I can remove the `@param`. Elsewhere, I'm implementing an interface, so I can't add PHP types on parameters.

Reply:
I've added a description to each parameter.